com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.Creator.cs

```diff
@@ -0,0 +1,101 @@
+using Chartboost.Banner;
+using Chartboost.Utilities;
+using UnityEditor;
+using UnityEngine;
+using static Chartboost.Utilities.Constants;
+
+namespace Chartboost.AdFormats.Banner.Unity
+{
+    public partial class ChartboostMediationUnityBannerAd : MonoBehaviour
+    {
+#if UNITY_EDITOR
+        [MenuItem("Chartboost Mediation/UnityBannerAd/Create New")]
+        [MenuItem("GameObject/Chartboost Mediation/UnityBannerAd")]
+        public static void CreateAd()
+        {
+            Instantiate();
+        }
+#endif
+
+        internal static ChartboostMediationUnityBannerAd Instantiate(
+            ChartboostMediationBannerAdSize size = null,
+            ChartboostMediationBannerAdScreenLocation screenLocation = ChartboostMediationBannerAdScreenLocation.Center)
+        {
+            // Find canvas with highest sorting order
+            var canvas = ChartboostMediationUtils.GetCanvasWithHighestSortingOrder();
+
+            // Instantiate inside this canvas
+            var unityBannerAd = new GameObject("ChartboostMediationUnityBannerAd")
+                .AddComponent<ChartboostMediationUnityBannerAd>();
+
+            unityBannerAd.transform.parent = canvas.transform;
+            unityBannerAd.name = "CM_UnityBannerAd";
+            unityBannerAd.transform.localScale = Vector3.one;
+            var rectTransform = unityBannerAd.gameObject.AddComponent<RectTransform>();
+            rectTransform.anchoredPosition = Vector2.zero;
```

Comment:
```suggestion
            unityBannerAd.name = "CM_UnityBannerAd";
            var bannerTransform = unityBannerAd.transform;
            bannerTransform.parent = canvas.transform;
            bannerTransform.localScale = Vector3.one;
            var rectTransform = unityBannerAd.gameObject.AddComponent<RectTransform>();
            rectTransform.anchoredPosition = Vector2.zero;
```

It's not ideal to always call `.transform` it's better to encapsulate on a variable and call whatever you need from it.